src/components/Login/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from "react";
+import { Form, Input, Button, Checkbox, Alert } from "antd";
+import { useHistory } from "react-router-dom";
+import firebase from "../../firebaseConfig";
+import { MailOutlined } from "@ant-design/icons";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+
+const layout = {
+  labelCol: { span: 4 },
+  wrapperCol: { span: 8 },
+};
+
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+const SignUp = ({ isTesting }) => {
```

Comment:
the directory name is Login why there's a signup component in here ?

Reply:
changed.